pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -2161,7 +2161,9 @@ class Build {
                                     def javaVersion = getJavaVersionNumber()
                                     if (!DEFAULTS_JSON['exclude-openjceplus'].contains(javaVersion)) {
                                         def openjceplusBranch = "semeru-java" + javaVersion.toString()
-                                        if (buildConfig.RELEASE) {
+                                        if(buildConfig.OPENJCEPLUS_BRANCH){
```

Comment:
Nit: Can you add a space after `if` and between `){` to be consistent formatting?